expression/builtin_op_vec_test.go

```diff
@@ -97,6 +99,7 @@ func makeGivenValsOrDefaultGener(vals []interface{}, eType types.EvalType) *give
 }
 
 func makeBinaryLogicOpDataGeners() []dataGenerator {
+	// TODO: rename this to makeBinaryOpDataGenerator, since the BIT ops are also using it?
```

Comment:
I don't think so.  `LogicOp` must uses the dataGeners to check the result is right. But for `BIT ops` ,it is unnecessary. Arbitrary int64 is the test case, and the data that the dataGeners gens  is the subset.

Reply:
Ack, how about we change this to say `TODO: remove this generator from the bit ops`?